Do not keep the u-boot environment stored on any media. Keeping it in the u-boot binary itself
helps us cope with flashing from SD card to internal media without worrying about changes to boot
parameters that may have been tampered with before.

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Jon Grisky <jon@resin.io>

Index: git/include/configs/mx6qvab820_common.h
===================================================================
--- git.orig/include/configs/mx6qvab820_common.h
+++ git/include/configs/mx6qvab820_common.h
@@ -264,7 +264,8 @@
 /* steven: use SPIROM, defined from boards.cfg */
 /* #define CONFIG_SYS_USE_SPINOR */ 
 /* steven: environment is in SPIROM */
-#define CONFIG_ENV_IS_IN_SPI_FLASH
+// #define CONFIG_ENV_IS_IN_SPI_FLASH
+#define CONFIG_ENV_IS_NOWHERE
 /* #define CONFIG_ENV_IS_IN_MMC */
 
 #ifdef CONFIG_CMD_SATA
